solr/core/src/java/org/apache/solr/handler/sql/SolrFilter.java

```diff
@@ -278,6 +265,24 @@ protected String translateComparison(RexNode node) {
       }
     }
 
+    private String toEqualsClause(String key, RexLiteral value, RexNode node) {
+      SqlTypeName fieldTypeName = ((RexCall) node).getOperands().get(0).getType().getSqlTypeName();
+      String terms = toSolrLiteralForEquals(value, fieldTypeName).trim();
+
+      boolean wrappedQuotes = false;
+      if (!terms.startsWith("(") && !terms.startsWith("[") && !terms.startsWith("{")) {
+        terms = "\"" + terms + "\"";
+        wrappedQuotes = true;
```

Comment:
Out of scope here, but #236 has me thinking about what if there are quotes inside of the terms.

Reply:
if terms has a quote, the query breaks ... I think we should just fix this here vs. opening another PR though

Reply:
@madrob This is a good catch and I just fixed in this PR but could break out to its own if you feel strongly about not fixing it here ;-)

Reply:
I was imagining that it would be more work than it ended up actually being